pkg/kustomize/kustomize_cli.go

```diff
@@ -10,16 +15,18 @@ import (
 // KustomizeCLI implements common kustomize actions based on kustomize CLI
 type KustomizeCLI struct {
 	Runner util.Commander
+	CWD    string
 }
 
 // NewKustomizeCLI creates a new KustomizeCLI instance configured to use the provided kustomize CLI in
 // the given current working directory
-func NewKustomizeCLI() *KustomizeCLI {
+func NewKustomizeCLI(cwd string) *KustomizeCLI {
```

Comment:
I don't think it is a good idea to set the working directory in the constructor. If you need to change the current working directory you would probably be better off to pass it as a parameter to the function which requires it. See _git_cli_ for example. One parameter to most of its functions is the directory of the repo.